std/string.d

```diff
@@ -4153,7 +4154,7 @@ unittest
         $(LREF entab)
   +/
 auto entabber(Range)(Range r, size_t tabSize = 8)
-    if (isForwardRange!Range)
+    if (isForwardRange!Range && isSomeChar!(ElementEncodingType!Range))
```

Comment:
Is this additional constraint necessary? It wasn't enforced before (and the doc doesn't mention it), but I also don't find any unittests that actually run the rangified functions on ranges that are not strings...